### PR TITLE
nav2_msgs readme

### PR DESCRIPTION
**Pull request by SteveMacenski**
I figure it doesnt hurt to give folks a little bit of help in the nav2_msgs readme

### CODE REVIEW

File: nav2_msgs/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# nav2_msgs
+
+The `nav2_msgs` package is a set of messages, services, and actions for the `navigation2` stack. The `navigation2` stack still makes use of `nav_msgs` from ROS (1) Navigation.
+
+The usage of messages and services in ROS 2 is slightly different. See a comparison below:
+
```
[the remark this answers is not in the document]
Comment (SteveMacenski) [reply nav2_msgs/README.md]: done

**Comment by bpwilcox** [comment]
The ROS2 wiki already has a section on messages/services in their [Migration Guide](https://index.ros.org/doc/ros2/Migration-Guide/#messages-and-services). I'd rather we link users to section of the guide and focus this README on what messages/services we are specifically adding in nav2_msgs and why.

**Comment by SteveMacenski** [comment]
Fair enough

**Comment by SteveMacenski** [comment]
This is now a bunch of commits for something small - this should be squash merged

**Pull request merged**